### PR TITLE
Fixes parse_netrc not grabbing passwords with spaces.

### LINKED ISSUE #23: Reading from .netrc only gets the first word of a multi-word password
For example, if my password is "aaa bbb ccc", it will be read as "aaa".  The same is true of user names, but I've never seen a bb user name of more than one word.  Note that prompting for the user name/password does not have this issue.
Example

### PR DESCRIPTION
**Pull request by AstralSorcerer**
Closes #23.

### CODE REVIEW

**Comment by joeljk13** [comment]
It may be a good idea to squash all these together as one commit - there's only really 1 change, not 5. But it's not that important, and I'd say the actual content is ready to merge.

**Comment by clehner** [comment]
+1 squash - we should probably get in the habit of doing that more (when merging into master).

Can the two sed commands be combined into one? i.e. `sed '/%TOKEN%/!d; regex1; regex2'`

**Comment by AstralSorcerer** [comment]
Good idea...I'm still a bit of a git noob.  Can I do that without having to do a seperate pull request?  And I think the sed commands can be combined, but only if the ```-n``` option is removed.

**Comment by clehner** [comment]
Yes, if you push (including with `--force`) to this branch, the pull req will update. See https://github.com/RocHack/bb/pull/21#issuecomment-70446509 for some info about squashing commits.
Yes, starting the sed command with `/.../!d` removes the need for `-n`.

**Comment by AstralSorcerer** [comment]
OK, will do!

**Comment by AstralSorcerer** [comment]
That what you were looking for?

**Comment by clehner** [comment]
You've got some merge conflict thingies in there but it looks like the regex is right. Consider also breaking it out it onto multiple lines if possible.

**Comment by AstralSorcerer** [comment]
The regex?  And I'm not seeing any merge conflicts.

File: bb [hunk 1]
```diff
@@ -69,8 +71,13 @@ parse_netrc() {
...
-	user_id=`sed -n 's/.*login \([^ ]*\).*/\1/p' <<< $cred`
-	password=`sed -n 's/.*password \([^ ]*\).*/\1/p' <<< $cred`
+<<<<<<< HEAD
+	user_id=`sed "${regex//%TOKEN%/login}" <<< $cred`
+	password=`sed "${regex//%TOKEN%/password}" <<< $cred`
+=======
+	user_id=`sed -n "${regex1//%TOKEN%/login}" <<< $cred | sed "$regex2"`
+	password=`sed -n "${regex1//%TOKEN%/password}" <<< $cred | sed "$regex2"`
+>>>>>>> b65da74... Fixed netrc regexs to properly handle backslash escapes
 }
```
Comment (clehner) [inline bb]: :point_up:

**Comment by AstralSorcerer** [comment]
Oh whoops.  I must have missed that.  Sorry.

**Comment by AstralSorcerer** [comment]
That better?

**Comment by clehner** [comment]
Right. breaking the regex can be done by replacing a semicolon or 2 with newlines (in `regex='...'`). what you did is also fine.

**Comment by AstralSorcerer** [comment]
OK.  Thanks for all the help!

**Comment by clehner** [comment]
You're welcome!

**Pull request merged**